synapse/res/templates/registration_token.html

```diff
@@ -1,12 +1,8 @@
 {% extends "_base.html" %}
 {% block title %}Authentication{% endblock %}
 
-{% block header %}
-<link rel="stylesheet" href="/_matrix/static/client/register/style.css">
-{% endblock %}
-
 {% block body %}
-<form id="registrationForm" method="post" action="{{ myurl }}">
```

Comment:
In `terms.html`, we define a style `registrationForm`. Do we want to do the same here or the default style is ok ?
Beware that I am mostly clueless regarding HTML/CSS.

Visually it looks good to me so perhaps not needed ?